extensions/default/src/utils/getSameSeriesOptions.ts

```diff
@@ -0,0 +1,38 @@
+import { DisplaySetService } from '@ohif/core';
+
+import getNextSRSeriesNumber from './getNextSRSeriesNumber';
+
+export default function getSameSeriesOptions(
+  SeriesDescription: string,
+  displaySetService: DisplaySetService
+): Record<string, unknown> {
+  const activeDisplaySets = displaySetService.getActiveDisplaySets();
+  const srDisplaySets = activeDisplaySets.filter(ds => ds.Modality === 'SR');
```

Comment:
When I read the function name i didn't get the idea it is SR only, please rename this function